net/nginx/Config_ssl.in

```diff
@@ -175,6 +175,11 @@ config NGINX_HTTP_V2
 	prompt "Enable HTTP_V2 module"
 	default y
 
+config NGINX_HTTP_QUIC
+	bool
+	prompt "Enable QUIC support"
```

Comment:
What is the size difference when enabled by default?

Reply:
Also:

> This is an experimental version and isn't fully supported because nginx
is being built with the regular OpenSSL and the regular one doesn't
support quic.

Soooo, why it is enabled by default?

Reply:
Fair point, It wasn't on purpose, corrected.